x-pack/plugins/security_solution/server/search_strategy/timeline/factory/events/all/helpers.test.ts

```diff
@@ -290,7 +267,7 @@ describe('#formatTimelineData', () => {
     };
 
     expect(
```

Comment:
I think the await goes outside the expect? https://jestjs.io/docs/en/tutorial-async#asyncawait

Reply:
I _think_ thats for with `.resolves`

In the first example:
```
it('works with async/await', async () => {
  expect.assertions(1);
  const data = await user.getUserName(4);
  expect(data).toEqual('Mark');
});
```
so mine is like this, but instead of using a variable `data` i put the await function call directly in the `()` because it will run what is inside the `()` before evaluating the expect